### PR TITLE
way-cooler: 0.6.2 -> 0.7.0

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/window-managers/way-cooler/default.nix [hunk 1]
```diff
@@ -9,65 +9,67 @@ let
   fakegit = writeShellScriptBin "git" ''
     echo ""
   '';
-  way-cooler = ((callPackage ./way-cooler.nix {}).way_cooler_0_6_2.override {
+  lua = lua5_3.overrideAttrs (oldAttrs: rec {
+    postFixup = ''
+      ln -s $out/lib/liblua.so $out/lib/liblua5.3.so
```
Comment (grahamc) [inline pkgs/applications/window-managers/way-cooler/default.nix]: This seems weird. Why not just patch lua to do this itself always?

Comment (gnidorah) [reply pkgs/applications/window-managers/way-cooler/default.nix]: @grahamc Fixed, thanks!

File: pkgs/applications/window-managers/way-cooler/default.nix [hunk 2]
```diff
@@ -9,65 +9,62 @@ let
...
+  way-cooler = ((callPackage ./way-cooler.nix {}).way_cooler_0_7_0.override {
     crateOverrides = defaultCrateOverrides // {
 
-    way-cooler = attrs: { buildInputs = [ wlc cairo libxkbcommon fakegit ]; };
+    way-cooler = attrs: { buildInputs = [ wlc cairo libxkbcommon fakegit gdk_pixbuf lua5_3 ]; };
     dbus = attrs: { buildInputs = [ pkgconfig dbus_libs ]; };
     gobject-sys = attrs: { buildInputs = [ dbus_glib ]; };
+    gio-sys = attrs: { buildInputs = [ dbus_glib ]; };
+    gdk-pixbuf-sys = attrs: { buildInputs = [ dbus_glib ]; };
+    gdk-pixbuf = attrs: { buildInputs = [ dbus_glib gdk_pixbuf ]; };
```
Comment (ne-sted) [inline pkgs/applications/window-managers/way-cooler/default.nix]: Is it a good idea to specify this in `default-crate-overrides.nix` instead?

Comment (gnidorah) [reply pkgs/applications/window-managers/way-cooler/default.nix]: @ne-sted Indeed, thanks!

**Pull request closed without merging**